test/box-luatest/gh_6539_log_user_space_empty_or_nil_select_test.lua

```diff
@@ -108,20 +112,24 @@ for _, eng in pairs{'memtx', 'vinyl'} do
         for _, call_fmt in pairs(explicit_fullscan_call_fmts) do
             local call = call_fmt:format(space)
             g.server:eval(call)
-            t.assert_not(g.server:grep_log(expected_log_entry, grep_log_bytes),
-                        ('log must not contain a critical entry about ' ..
-                         'an empty or nil `select` call on a %s '..
-                         'user space with `fullscan = true` option '..
-                         'set'):format(eng))
+            t.helpers.retrying({}, function()
+                t.assert_not(g.server:grep_log(expected_log_entry, grep_log_bytes),
+                             ('log must not contain a critical entry about ' ..
+                              'an empty or nil `select` call on a %s '..
+                              'user space with `fullscan = true` option '..
+                              'set'):format(eng))
```

Comment:
Nit: while you are here, would be nice to concatenate all these strings.

> However, never break user-visible strings such as printk messages, because that breaks the ability to grep for them. [^1]

[^1]: https://www.kernel.org/doc/html/v4.10/process/coding-style.html#breaking-long-lines-and-strings